orchestrator/go.mod

```diff
@@ -5,8 +5,11 @@ go 1.23
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.12.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.6.0
+	github.com/Azure/azure-sdk-for-go/sdk/security/keyvault/azcertificates v1.1.0
 	github.com/SanteonNL/go-fhir-client v0.2.7
 	github.com/SanteonNL/nuts-policy-enforcement-point v0.0.0-20240805130644-8b92f1cbf865
+	github.com/beevik/etree v1.2.0
+	github.com/braineet/saml v0.4.15
```

Comment:
we still using this, or can it go?

Reply:
We are, `etree` for the xml elements we create and `braineet/saml` is used for it's `xmlenc` package